src/main/java/labs/introtoprogramming/lab5/gui/Input.java

```diff
@@ -0,0 +1,59 @@
+package labs.introtoprogramming.lab5.gui;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+public class Input {
+
+  private boolean forwardKeyDown = false;
```

Comment:
Assignment is redundant. Is it made for clarity?

Reply:
Yes, to make it clear that my default all keys are not down.

Reply:
Yes, to make it clear that all keys are not down by default.